components/ArticleReplyFeedbackControl/ReasonsDisplay.js

```diff
@@ -95,9 +95,21 @@ function ReasonsDisplay({ articleReply, onSizeChange = () => {} }) {
     if (onSizeChange) return onSizeChange();
   }, [tab, onSizeChange]);
 
+  const IsEmptyComment = comment => comment === '' || comment === null;
+
   const feedbacks =
     data?.ListArticleReplyFeedbacks?.edges.map(({ node }) => node) || [];
 
+  const processedFeedbacks = (feedbacks, voteType, isLoadMore) =>
```

Comment:
nit: we can also move this processor function to outside of the component to avoid re-creating the function on everyrender.